android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1504,6 +1504,21 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_GROUP_KEY = "groupKey";
+
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_GROUP_ALERT_BEHAVIOUR = "groupAlertBehaviour";
```

Comment:
Our APIs use American English spelling rules. So, we should rename "Behaviour" to "Behavior", without the 'u'. (Sorry)

This would match our spelling with iOS' notification behavior property...
http://docs.appcelerator.com/platform/latest/#!/api/Titanium.App.iOS.UserNotificationAction-property-behavior